lib/devices.ts

```diff
@@ -190,6 +190,10 @@ export class BalenaFin extends DeviceInteractor {
 	// usb-toggle
 	async toggleUsb(state: boolean, port: number) {
 		console.log(`Toggling USB ${state ? 'on' : 'off'}`);
+		// we will do this twice, as there could be issues with the driver that cause the hub to re-power the device (https://github.com/balena-io-hardware/balena-fulfillment-rig/blob/master/core/app/utils/dut-power/index.js#L26)
```

Comment:
Can we create an issue somewhere to track this? Pointing at a comment in code could get lost if there's a refactor :)

Reply:
yep will make the issue

Reply:
uhubctl author here. Instead of doing this, I recommend to add `-r 1000` option instead

Reply:
Thanks for the suggestion @mvp ! Adding this as a note to #27 @rcooke-warwick